### ISSUE TITLE
The training loss is contain 1.33, it can not decrease.

### ISSUE THREAD

**Issue by 201814476285951** [xingyizhou/CenterNet#243]
I used python main.py --task ctdet --exp_id coco_dla --batch_size 32 --master_batch 15 --lr 1.25e-4^C--gpus 0,1
The datasets have not changed.

**Comment by xingyizhou** [comment]
It does take long to train coco in two GPUs. You can see what happens after training it for a day.

**Comment by 201814476285951** [comment]
It have trained 40 hours,running to about 40 epoch ,and it still contain 1.32

**Comment by xingyizhou** [comment]
Can you make sure the code is unchanged? Or you can try training pascal voc first to see if you can reproduce the results.

**Comment by Holmes-GU** [comment]
Hi, when you finish raining, about how larg the hm_loss drops to?
